### DISCUSSION TITLE
Some shortcuts to consider

### DISCUSSION THREAD

**Discussion by uahnbu** [kimlimjustin/xplorer#191] [Ideas]
These are some of my mostly used and wanted shortcuts to help with the Xplorer flow
1. `Backspace` to go back to previous folder (like in Chrome history, not going up the directory tree)
2. `Ctrl/Cmd + O` to close the preview pane (if the pane is already opened)
3. Press letter keys to search for file/folder name starting with these letters. The search string reset 500ms after no more letter key is pressed.

**Comment by kimlimjustin** [comment]
> `Backspace` to go back to previous folder (like in Chrome history, not going up the directory tree)

What backspace key do you mean? the mouse key? or keyboard? also, to go back to preview folder, there is `Alt + Left` shortcut for now.

> `Ctrl/Cmd + O` to close the preview pane (if the pane is already opened)

Implemented! Will also implement auto-update the preview pane according to the file selected

> Press letter keys to search for file/folder name starting with these letters. The search string is reset 500ms after no more letter key is pressed.

Nice idea, will implement soon!

**Reply by uahnbu** [reply]
Oh `Alt+Left` makes more sense. I use `Backspace` key in the native File Explorer but `Alt+Left` is more of a match with Chrome's behavior. Also I've just discovered the shortcuts documentation page 🤣

**Reply by kimlimjustin** [reply]
Oh, just know `Backspace` is a key to go back in native file explorer, does the shortcut apply in other OS too? if it does, maybe I'll add it

**Reply by uahnbu** [reply]
Maybe it's just Windows. I've just read about Ubuntu as in [here](https://ubuntuforums.org/showthread.php?t=2182910) and [here](https://askubuntu.com/questions/398984/backspace-shortcut-for-explorer) (from 2014) but maybe they only use `Alt+Left`.

**Reply by kimlimjustin** [reply]
I think maybe I'll just add the feature to customize shortcuts in the future (maybe the end of this month when I got time)

**Reply by uahnbu** [reply]
That's great news to hear. I'm looking forward to more customizations 🥳

**Reply by kimlimjustin** [reply]
Oh yeah, if you want to, any PR is welcome :)

**Reply by kimlimjustin** [reply]
> Press letter keys to search for file/folder name starting with these letters. The search string is reset 500ms after no more letter key is pressed.

Implemented!

**Reply by uahnbu** [reply]
That's such a rapid response :smile: I'm glad to have spent an hour to make Tauri available on my computer to be able to test the latest features :rofl: Tbh I've read your comment since yesterday but I've had so many things to add that I delayed. I'll post them in a new discussion as it's getting off-topic.
